SystemSettings.php

```diff
@@ -53,11 +53,14 @@ class SystemSettings extends \Piwik\Settings\Plugin\SystemSettings
     public $numRequestsToProcess;
 
     /** @var Setting */
-    public $useSentinelBackend;
+    public $useWhatRedisBackendType;
 
     /** @var Setting */
     public $sentinelMasterName;
 
+    /** @var Array */
+    protected $availableRedisBackendType = array(1=>"Stand Alone", 2=>"Sentinel", 3=>"Cluster");    
```

Comment:
It's a bit of a hassle, but these 3 strings should probably be translations in lang/en.json. You can use `Piwik::translate()` to translate each element while defining the array.

Reply:
at first i thought these 3 strings defined a type of product that could not be translated freely. just imagining that when the word "Sentinel" is translated into indonesian as "Penjaga", i think it will create a little confusion regarding the definition of Redis Sentinel. but i'll fix it anyway and use `Piwik::translate()`, thank you..

Reply:
this is fixed, can you confirm?

Reply:
Looks good. The only things that I'd potentially change would be:
1. Make the new method name plural, like `getAvailableRedisBackendTypes` instead of `getAvailableRedisBackendType`.
2. I wonder if the translation string for `AvailableRedisBackendTypeStandAlone` should be `Stand-alone`.

Reply:
sure.. fixed.. 
thanks👍

Reply:
Thank you. Looks good :+1: